### PR TITLE
Schedule lambdas to run daily

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -113,3 +113,12 @@ Resources:
             Timeout: 100
         DependsOn: LambdaRole
 
+    Scheduler:
+        Type: AWS::Events::Rule
```
Comment (hosamaly) [inline cloudformation.yaml]: Do we need to give the scheduler permission to invoke the lambda? Something like this?
```yaml
  SchedulerLambdaPermission:
    Type: AWS::Lambda::Permission
    Properties:
      Action: lambda:InvokeFunction
      FunctionName: !GetAtt UserReminderLambda.Arn
      Principal: events.amazonaws.com
      SourceArn: !GetAtt Scheduler.Arn
```

Comment (regiskuckaertz) [reply cloudformation.yaml]: Yeah, looks like it is needed. What a waste of time and energy 💩

Comment (hosamaly) [reply cloudformation.yaml]: The Serverless Transform does it automatically 😉

Comment (regiskuckaertz) [reply cloudformation.yaml]: Done, thanks for your advice 👍

Comment (regiskuckaertz) [reply cloudformation.yaml]: Ooooh, so I might take up on your offer and do it in a separate PR (I did not see the advantage but if it's cleaner and easier to reason about, then 👍 )

**Comment by regiskuckaertz** [comment]
Yes, but will do in a separate PR.

File: cloudformation.yaml [hunk 2]
```diff
@@ -113,3 +113,28 @@ Resources:
...
+        Properties:
+            ScheduleExpression: cron(12 14 * * * *) # run daily at 14:13
+            Targets:
+                - Arn: !Ref UserReminderLambda
+                  Input: "true" # required to perform a full run
+                - Arn: !Ref DidNotAnswerLambda
+                  Input: "true" # required to perform a full run
+
+    UserRemminderPermission:
```
Comment (hosamaly) [inline cloudformation.yaml]: There is an extra `m` here.

Comment (regiskuckaertz) [reply cloudformation.yaml]: 🦅 eye

**Pull request merged**